<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rathinavel" failures="2" tests="2" name="Test(failed)" time="81.929" errors="0" timestamp="2025-01-15T00:24:24 IST">
  <testcase classname="testCases.TC004_ValidateOrangeHRMDashboard" name="validateDashboard" time="21.186">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at testCases.TC004_ValidateOrangeHRMDashboard.validateDashboard(TC004_ValidateOrangeHRMDashboard.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- validateDashboard -->
  <testcase classname="testCases.TC003_LoginOrangeHRMTest" name="loginOrangeHRM" time="21.087">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at testCases.TC003_LoginOrangeHRMTest.loginOrangeHRM(TC003_LoginOrangeHRMTest.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginOrangeHRM -->
</testsuite> <!-- Test(failed) -->
